### ISSUE TITLE
2020.6 update

### ISSUE THREAD

**Issue by StevenWilliams** [nlordell/mullvad-vpn-ostree#1]
Tried changing version number to 2020.6.

Got "Cannot move opt/Mullvad VPN to usr/share/mullvad-vpn/Mullvad VPN": Directory not empty

Also got a similar error for zsh, which was resolved by commenting usr/local/share/zsh usr/share/zsh as the package already had the zsh directory in usr/share.

Tried building on the fedora 33 in toolbox. Got a warning for could not cannonicalize hostname: toolbox, but this isn't where it fails

**Comment by nlordell** [comment]
Yeah, I think it the RPM changed the directory structure a bit in version 2020.6 that probably requires some adjustments to the spec.

**Comment by nlordell** [comment]
I've been meaning get this spec building from source but I just haven't found the time 😢
